src/Schema/Table.php

```diff
@@ -847,6 +850,31 @@ public function getComment(): ?string
         return $this->_options['comment'] ?? null;
     }
 
+    /**
+     * Instantiates a new table editor.
+     */
+    public static function editor(): TableEditor
+    {
+        return new TableEditor();
+    }
+
+    /**
+     * Instantiates a new table editor and initializes it with the table's properties.
+     */
+    public function edit(): TableEditor
+    {
+        return self::editor()
+            ->setName($this->getObjectName()->toString())
```

Comment:
Even though `getObjectName()` can throw `NameIsNotInitialized`, this isn't a BC break because it's a new method.